build.gradle

```diff
@@ -16,5 +17,6 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        google()
```

Comment:
What is this for?

Reply:
Diego this is needed for support library we are using from google maven repository i.e. `google()`. The latest versions and future support for android related libraries now will be used from google maven repository in gradle 4+ otherwise it will give me build error. More on here https://developer.android.com/studio/build/dependencies.html#google-maven

Reply:
Thanks for the explanation and the link!